jobs/config.json

```diff
@@ -9529,13 +9529,24 @@
   "pull-kubernetes-verify": {
     "args": [
       "--branch=${PULL_BASE_REF}",
+      "--jenkins",
```

Comment:
you need to add `--jenkins` to ci-verify jobs as well I think

Reply:
Or maybe change it to `--prow` so you won't mess up with existing jobs

Reply:
hmm yeah, let me flip it to `--prow` for now

Reply:
done